tests/utils.go

```diff
@@ -538,17 +538,17 @@ func CleanNodes() {
 
 		old, err := json.Marshal(node)
 		Expect(err).ToNot(HaveOccurred())
-		new := node.DeepCopy()
+		newNode := node.DeepCopy()
 
 		k8sClient := GetK8sCmdClient()
 		if k8sClient == "oc" {
-			RunCommandWithNS("", k8sClient, "adm", "uncordon", node.Name)
+			_, _, _ = RunCommandWithNS("", k8sClient, "adm", "uncordon", node.Name)
 		} else {
-			RunCommandWithNS("", k8sClient, "uncordon", node.Name)
+			_, _, _ = RunCommandWithNS("", k8sClient, "uncordon", node.Name)
 		}
 
 		found := false
-		taints := []k8sv1.Taint{}
+		var taints []k8sv1.Taint
```

Comment:
in a separate commit you could teach me why this is better.

Reply:
Will split.

The only real difference is that it saves an allocation. The slice will stay nil as long as nothing gets appended to it.
There seems to be an agreement online that it's better to initialize slices that way.
This is not far from a religious debate, but the former does pop a warning in goland!